__tests__/bubbleNub.js

```diff
@@ -43,16 +45,18 @@ describe('bubble nub', () => {
   beforeEach(() => {
     props = {
       styleOptions: {
-        bubbleBorderColor: 'red',
```

Comment:
We can't remove`bubbleBorder*` from the style options because customers might be using these options. Removing them would be a breaking change. Instead set the default value to `undefined`  and in the create style set function use `bubbleBorder*` if it's provided; otherwise, use `bubbleFromBotBorder*`